crates/subspace-service/src/domains.rs

```diff
@@ -112,12 +114,19 @@ where
         &self,
         block_hash: Option<CBlock::Hash>,
     ) -> Option<ExecutionReceiptFor<Block::Header, CBlock, Balance>> {
+        const ATTEMPTS_NUMBER: u32 = 5;
+        const PEERS_THRESHOLD: usize = 5;
```

Comment:
I moved the constants and increased their values for now, however, it seems we'll change either their values or meaning because of the issue found by Ning in the comments below.